src/app/pipes/milestone.pipe.ts

```diff
@@ -530,13 +534,15 @@ export class MilestonePipe implements PipeTransform {
           case '2022-12-13':
             return ' | The Dawning'
           case '2022-12-20':
-            return ' | Exotic Quest: Division Zero'
+            return ' | Exotic Quest: Revision Zero'
```

Comment:
😂

Reply:
Was probably me typing it out at like 1am 😞